crates/node-file-trace/src/lib.rs

```diff
@@ -487,24 +487,27 @@ async fn run<B: Backend + 'static, F: Future<Output = ()>>(
     let (sender, mut receiver) = channel(1);
     let dir = current_dir().unwrap();
     let tt = create_tt();
-    let console_ui = Arc::new(ConsoleUi::new(LogOptions {
-        current_dir: dir.clone(),
-        show_all,
-        log_detail,
-        log_level: log_level.map_or_else(|| IssueSeverity::Error, |l| l.0),
-    }));
     let task = tt.spawn_root_task(move || {
+        let console_ui = ConsoleUiVc::new(TransientInstance::new(LogOptions {
```

Comment:
This is the bug. The `TransientInstance` needs to be extracted out and shared by all calls, that way the same `ConsoleUiVc` will be reused.